hydro_deploy/core/src/ssh.rs

```diff
@@ -86,9 +89,40 @@ impl Drop for LaunchedSSHBinary {
         let session = self.session.take().unwrap();
         std::thread::scope(|s| {
             s.spawn(|| {
-                let runtime = tokio::runtime::Builder::new_multi_thread().build().unwrap();
+                let runtime = tokio::runtime::Builder::new_multi_thread()
+                    .enable_time()
+                    .build()
+                    .unwrap();
                 runtime
-                    .block_on(session.disconnect(None, "", None))
+                    .block_on(async move {
+                        self.channel.write_all(&[b'\x03']).await.unwrap();
```

Comment:
We can probably move this into the block if we want perf data.

Reply:
wdym?

Reply:
Only send the Ctrl-C and wait for the program to shutdown if we launched the program with perf; otherwise the default logic of just dropping the connection should be fine.